src/download.c

```diff
@@ -149,7 +149,7 @@ int start_full_download(bool pipelining)
 	 * is even more broken than average. This at least will allow the user to update, albeit slowly.
 	 */
 	if (pipelining) {
-		curl_multi_setopt(mcurl, CURLMOPT_PIPELINING, 1);
+		curl_multi_setopt(mcurl, CURLMOPT_PIPELINING, CURLPIPE_MULTIPLEX | CURLPIPE_HTTP1);
```

Comment:
this should be called out as a change for libcurl compatibility (1 and 0 were used in versions prior to 7.43.0, and the bitmask is used in newer versions).